code/game/objects/structures/extinguisher.dm

```diff
@@ -17,11 +17,11 @@
 	var/opened = 0
 	var/material_drop = /obj/item/stack/sheet/metal
 
-/obj/structure/extinguisher_cabinet/New(turf/loc, ndir = null)
+/obj/structure/extinguisher_cabinet/New(turf/loc, direction = null)
 	..()
```

Comment:
You don't return the ..() here while you do in the subtype. Probably not required but it's inconsistent